src/services/migration/MigrationService.php

```diff
@@ -29,8 +29,10 @@ public function ensureFitness(): void {
     /**
      * @return Migration[]
      */
-    public function getAllMigration(): array {
-        return $this->factory->getMigrations();
+    public function getUserMigrations(): array {
+        return \array_filter($this->factory->getMigrations(), function (Migration $migration) {
+            return !$migration->mustMigrate();
```

Comment:
Instead of "abusing" the `mustMigrate` method here, we should add an explicit `isUserMigration`. Or, in case we don't have a use of the `mustMigrate` method rename it.

Reply:
It also prevent the case of mandatory user oriented migration. Fixed